src/genie/libs/parser/asa/show_crypto_ikev2_sa.py

```diff
@@ -0,0 +1,220 @@
+"""
+show_crypto_ikev2_sa.py
+Parser for the following command:
+    * show crypto ikev2 sa
+"""
+import re
+
+from genie.metaparser import MetaParser
+from genie.metaparser.util.schemaengine import (Schema, Any, Optional)
+
+
+class ShowCryptoIkev2SaSchema(MetaParser):
+    """Schema for
+        * show crypto ikev2 sa
+    """
+
+    schema = {
+        'sessions': {
+            Any(): {
+                'session_id': int,
```

Comment:
I could be mistaken, but I believe this key can be removed as session ID is already used as dictionary key

![image](https://user-images.githubusercontent.com/33130026/119907924-64257580-bf1f-11eb-9eb1-1b2aa4463198.png)